test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
- [ ] The test.md file should not be present in your remote repository. To remove it from your repository please refer to the this [stack overflow](https://stackoverflow.com/questions/2047465/how-do-i-delete-a-file-from-a-git-repository) thread.